portfolio/src/main/java/com/google/sps/data/User.java

```diff
@@ -0,0 +1,40 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.data;
+
+public class User //this object represents each comment posted to the website {Name: Comment}
+{
+    private long id;
+    private String name;
+    private String message;
+
+    public User(long id, String name, String message){
+        this.id = id;
+        this.name = name;
+        this.message = message;
+    }
+
+    public long getId(){
+        return id;
+    }
+
+    public String getName(){
+        return name;
+    }
+
+    public String getMessage(){
+        return message;
+    }
+}
```

Comment:
Are you sure you want a comments entity defined as User? If you are just adding Comments, I'd recommend just purely have a Comments class. If you want the name of the user, you can keep that but I think it would be better to hold off on doing user related work until you do the Authentication Walkthrough.  You can totally link comments to your own user defined entities as well but usually better to use IDs instead of name. That way you can separately add more fields to the user entity and still reference those fields from the comments entity without having to duplicate fields such as "name".

Reply:
Sorry, I forgot to add comments to describe why I structured the User class that way. I was going to store lists of comments associated with each user (based on their id) so that people could load up all the comments from a specific user if they wanted to. I see that my structure is kind of misleading. I'll use a Comment class for each individual message and then store lists of those Comment objects within each User class for the authentication portion of the project.

Reply:
When I implement the new User class holding the Comment objects I'll ensure the IDs match up so that I only have to store and display the name once